lib/tasks/prepare_postgres.rake

```diff
@@ -48,13 +55,27 @@ private
     end
   end
 
-  def package_missing?(package)
-    command = 'which ' + package
-
-    `#{command}`[0] != '/'
+  def exec_present? name
+    system("which #{name} 2>&1 >/dev/null")
   end
 
   def darwin_install_package(package)
```

Comment:
As we're moving away from a "one-click-installer", let's delete the method `darwin_install_package` to not have it slushing around perhaps?

Reply:
Ya, I had a problem when it tried to install postgres for me:

a) I already had `postgresql@9.6` installed, which matches the version in production, so I didn't really want the latest Postgres installed.
b) it hung for some reason and didn't complete, for some reason ... maybe brew doesn't like running that way? Not sure.

I agree we should probably just remove the method.

Reply:
I tested it, and it just gave an error message in the build log, but still succeeded - seems floppy, so better to remove it. Mac devs should know how to use brew, and if not, it's in the error msg.